data/sql/updates/pending_db_world/rev_1609870134525124000.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609870134525124000');
+UPDATE `spell_proc_event` SET `procFlags` = 279552, `procEx` = 786432 WHERE `entry` = 64411;
```

Comment:
Pls add flags like this `x`=`x`|Flag1|Flag2|Flag3 to easily see what flags have been added